### PR TITLE
(WIP) Feat(UI)/SRT-4: Create initial build page

### PR DESCRIPTION
**Pull request by maymax777**
- Create initial build page template

### CODE REVIEW

File: src/css/v2/carousel.scss [hunk 1]
```diff
@@ -28,6 +28,16 @@
...
   }
+
+  &-bg-build-1 {
+    background-image: url(../images/v2/carousel-bg-build-1.svg);
+    background-repeat: no-repeat;
+    background-position: right;
+    background-size: 40% 40%;
+    background-color: rgba(244, 244, 244, 0.8);
+    background-blend-mode: multiply;
+    // background-position-x: 293px;
```
Comment (micahjlucas) [inline src/css/v2/carousel.scss:39]: Please remove or add comment message.

File: src/pages/v2/build.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+template: v2-build-page
```
Comment (micahjlucas) [inline src/pages/v2/build.md:2]: Should probably remove v2 here, since that directory is removed.

File: src/templates/v2-build-page.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import Carousel from "../components/v2/Carousel"
+import Header from "../components/v2/Header"
+import Footer from "../components/v2/Footer"
+
+const carousel1 = {
```
Comment (micahjlucas) [inline src/templates/v2-build-page.js:6]: Please rename to something more descriptive - Example: `carouselBuild`

**Review by micahjlucas — CHANGES_REQUESTED** [review]
Code looks good overall, just added a couple comments.

**Comment by maymax777** [comment]
Duplicated from #367

**Pull request closed without merging**